Peek-swiftUI/Utilites/Extentions/View+FavoriteError.swift

```diff
@@ -0,0 +1,19 @@
+//
+//  View+FavoriteError.swift
+//  Peek-swiftUI
+//
+//  Created by mazen eldeeb on 29/08/2024.
+//
+
+import SwiftUI
+
+
+extension View {
+    public func favoriteError(isPresented: Binding<Bool>,
```

Comment:
why is this not used
if no need for it remove it

Reply:
to present any error resulting from coredata methods, used in multiple place